.vscode/settings.json

```diff
@@ -16,6 +16,67 @@
     "C_Cpp.dimInactiveRegions": true,
     "files.associations": {
         "string": "cpp",
-        "filesystem": "cpp"
+        "filesystem": "cpp",
+        "array": "cpp",
+        "atomic": "cpp",
+        "hash_map": "cpp",
+        "hash_set": "cpp",
+        "*.tcc": "cpp",
+        "bitset": "cpp",
+        "cctype": "cpp",
+        "chrono": "cpp",
+        "clocale": "cpp",
+        "cmath": "cpp",
+        "complex": "cpp",
+        "condition_variable": "cpp",
+        "cstdarg": "cpp",
+        "cstddef": "cpp",
+        "cstdint": "cpp",
+        "cstdio": "cpp",
+        "cstdlib": "cpp",
+        "cstring": "cpp",
+        "ctime": "cpp",
+        "cwchar": "cpp",
+        "cwctype": "cpp",
+        "deque": "cpp",
+        "list": "cpp",
+        "unordered_map": "cpp",
+        "unordered_set": "cpp",
+        "vector": "cpp",
+        "exception": "cpp",
+        "algorithm": "cpp",
+        "functional": "cpp",
+        "optional": "cpp",
+        "ratio": "cpp",
+        "string_view": "cpp",
+        "system_error": "cpp",
+        "tuple": "cpp",
+        "type_traits": "cpp",
+        "fstream": "cpp",
+        "initializer_list": "cpp",
+        "iomanip": "cpp",
+        "iosfwd": "cpp",
+        "iostream": "cpp",
+        "istream": "cpp",
+        "limits": "cpp",
+        "memory": "cpp",
+        "mutex": "cpp",
+        "new": "cpp",
+        "ostream": "cpp",
+        "numeric": "cpp",
+        "shared_mutex": "cpp",
+        "sstream": "cpp",
+        "stdexcept": "cpp",
+        "streambuf": "cpp",
+        "thread": "cpp",
+        "cinttypes": "cpp",
+        "regex": "cpp",
+        "utility": "cpp",
+        "typeinfo": "cpp",
+        "bit": "cpp",
+        "map": "cpp",
+        "iterator": "cpp",
+        "memory_resource": "cpp",
+        "random": "cpp"
     },
 }
```

Comment:
In general, this file should not need to be modified for this PR, IMO. I'd recommend creating a separate PR with all the vscode-related changes.

Reply:
I'd add that changes to .vscode should not be tracked period. Instead, each user gets a tracked .vscode_shared/UserName folder, and then this is linked to with a regenerate_vscode.sh script.

See for example: https://github.com/bHimes/cisTEM_downstream_bah/commit/22e4601ec34156b07ab449427c1cbfa597ac8ede

Note, this commit I point to isn't complete, but is where I made the most relevant changes. I can cherry pick those relevant to this approach and create a PR if you want. This is probably better to do before you commit any changes to .vscode files that I don't have.